server/gokbg3/grails-app/controllers/org/gokb/rest/CuratoryGroupsController.groovy

```diff
@@ -117,7 +119,8 @@ class CuratoryGroupsController {
     def errors = [:]
 
     if (group) {
-      def qry = "where exists (select arg from AllocatedReviewGroup arg where arg.group = :group and arg.review = rr)"
+      def qry = "where exists (select arg from AllocatedReviewGroup arg" +
+                              "where arg.group = :group and arg.review = rr and arc.status != :inactive)"
```

Comment:
The query param `:inactive` has not been defined here. Need to retrieve the RDV.